cmd/gateway/commands.go

```diff
@@ -123,6 +123,9 @@ func createStaticModeCommand() *cobra.Command {
 	configName := stringValidatingValue{
 		validator: validateResourceName,
 	}
+	var enableMetrics bool
+	var metricsListenPort int
```

Comment:
I looked at this when I was implementing the flag, but because there's a default value, I'm using `IntVar` which has a different signature from `Var` (it uses `*int` for the value and not `pflag.Value`). I'll take another look at this though.